cdap-client/src/main/java/co/cask/cdap/client/StreamClient.java

```diff
@@ -274,7 +273,7 @@ public void setTTL(Id.Stream stream, long ttlInSeconds)
   /**
    * Lists all streams.
    *
-   * @return list of {@link StreamRecord}s
+   * @return list of {@link StreamDetail StreamDetails}
```

Comment:
One is the link, the second is a label. As you can see, lets one use consistent pluralization.